less-discourse/menu.css

```diff
@@ -0,0 +1,31 @@
+body {
```

Comment:
I don't understand why you need this file?
Why not using those styles in ``less-discourse/discourse.less``?